xbmc/utils/JobManager.h

```diff
@@ -322,6 +322,7 @@ class CJobManager
 protected:
   friend class CJobWorker;
   friend class CJob;
+  friend class CJobQueue;
```

Comment:
Really? Why do you not introduce a public method `CJobMananger::IsRunning`?

Reply:
I almost did that but I didn't for a few reasons: 

1) The m_isrunning state isn't used in other places you might expect like JobManager::IsProcessing so it seems to be really in internal implementation detail. 
2) The JobQueue and the JobManager are already completely dependent on each others implementation details so knowing that the `JobManager::m_isrunning` flag, when set to `false` means the `JobQueue::IsProcessing` should be false didn't seem like it needed a change to the public interface.

If people would prefer it the other way, I have no problem changing it.

Reply:
Yeah, it's messy. I just thought "why make it even worse?"

Reply:
I think the point was the keep the mess internal rather than let it leak into the public interface that would need to be maintained going forward.

Reply:
No hard feelings here. As it is working as it is, it's your decision. :-)